node/lib/util/stitch_util.js

```diff
@@ -778,8 +778,15 @@ exports.readConvertedCommits = co.wrap(function *(repo) {
 
     const result =
                  yield BulkNotesUtil.readNotes(repo, exports.convertedNoteRef);
-    for (let key in result) {
-        result[key] = exports.readConvertedContent(result[key]);
+    for (let key of Object.keys(result)) {
```

Comment:
nil, `for (const [key, sha] of Object.entries(result))` is probably easier to read